talk/objectorientation/adl.tex

```diff
@@ -144,3 +144,27 @@
     \end{columns}
   \end{block}
 \end{frame}
+
+\begin{frame}[fragile]
+  \frametitlecpp[98]{ADL and Hidden Friends}
+  \begin{block}{Hidden friend declarations participate in ADL}
+    \begin{itemize}
+      \item Friends defined in class scope are ``hidden friends''
+      \item They are \emph{not} class members (no access to \mintinline{cpp}{this})
+      \item They participate in ADL, e.g.\ \mintinline{cpp}{std::cout << complex}
+    \end{itemize}
+  \end{block}
+  \begin{exampleblock}{Ostream operator as hidden friend}
```

Comment:
Can you please try to make the title more precise? You could also mean `operator>>` here. Maybe:
```suggestion
  \begin{exampleblock}{operator<< as hidden friend}
```
`operator<<` is sometimes also called the "put operator".